snippets/csharp/VS_Snippets_Remoting/Classic Socket.Connect Example/CS/source.cs

```diff
@@ -5,54 +5,56 @@
 
 public class Sample
 {
-  static void ConnectAndCheck(Socket client, EndPoint anEndPoint)
-  {
+    static void ConnectAndCheck(Socket client, EndPoint anEndPoint)
+    {
 // <Snippet1>
-    // .Connect throws an exception if unsuccessful
-    client.Connect(anEndPoint);
+        // .Connect throws an exception if unsuccessful
+        client.Connect(anEndPoint);
 
-    // This is how you can determine whether a socket is still connected.
-    bool blockingState = client.Blocking;
-    try
-    {
-        byte [] tmp = new byte[1];
+        // This is how you can determine whether a socket is still connected.
+        bool blockingState = client.Blocking;
+        try
+        {
+            byte [] tmp = new byte[1];
 
-        client.Blocking = false;
-        client.Send(tmp, 0, 0);
-        Console.WriteLine("Connected!");
-    }
-    catch (SocketException e) 
-    {
-        // 10035 == WSAEWOULDBLOCK
-        if (e.NativeErrorCode.Equals(10035))
-            Console.WriteLine("Still Connected, but the Send would block");
-        else
+            client.Blocking = false;
+            client.Send(tmp, 0, 0);
+            Console.WriteLine("Connected!");
+        }
+        catch (SocketException e) 
         {
-            Console.WriteLine("Disconnected: error code {0}!", e.NativeErrorCode);
+            // 10035 == WSAEWOULDBLOCK
+            if (e.NativeErrorCode.Equals(10035))
+            {
+                Console.WriteLine("Still Connected, but the Send would block");
+            }
+            else
+            {
+                Console.WriteLine("Disconnected: error code {0}!", e.NativeErrorCode);
```

Comment:
Future suggestion: We've been updating these to use interpolated strings in preference to the positional formatting.

Reply:
Thanks! I didn't want to change code yet in this PR, I will keep it in mind for future changes.